content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -1065,6 +1065,11 @@ status:
   terminating: 3 # three Pods are terminating and have not yet reached the Failed phase
 ```
 
+{{< note >}}
+Since v1.31 the `JobPodReplacementPolicy` feature gate guards the fix for the issue:
```

Comment:
We prefer to link to the list of feature gates:
```suggestion
Since v1.31 the `JobPodReplacementPolicy` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) guards the fix for the issue:
```

Reply:
For sure, it was just a quick note. I m going to work on a proper documentation update this week.

Reply:
Updated. PTAL